src/test/resources/sms.properties

```diff
@@ -1,11 +1,40 @@
-# Configuration file for SMS Sender
```

Comment:
I'm curious, what is the purpose of the empty `src/test/resources/language.properties` file just above this?

Reply:
I think garbage from previous tests